src/overlays/actors/ovl_Obj_Ice_Poly/z_obj_ice_poly.h

```diff
@@ -7,11 +7,18 @@ struct ObjIcePoly;
 
 typedef void (*ObjIcePolyActionFunc)(struct ObjIcePoly*, GlobalContext*);
 
+#define OBJICEPOLY_GET_FF00(thisx) (((thisx)->params >> 8) & 0xFF)
+
+#define OBJICEPOLY_FF_FF 0xFF
```

Comment:
Right before this got decomped I noticed this shows up a lot with actors that have switch flags, where it checks the max value of 0x7F or 0xFF, and if thats the case the variant ignores switch flags, like this actor here.

Starting to think there should be a general define somewhere instead like "SWITCH_FLAG_IGNORE" or something